addons/account/views/product_views.xml

```diff
@@ -16,14 +16,15 @@
          <field name="model">product.product</field>
          <field name="inherit_id" ref="product.product_product_view_form_normalized"/>
          <field name="arch" type="xml">
-             <xpath expr="//div[@name='sales_price']" position="inside">
-                 <field name="tax_string"/>
-             </xpath>
-             <xpath expr="//div[@name='sales_price']" position="after">
-                 <field name="taxes_id" widget="many2many_tags"
-                        context="{'default_type_tax_use': 'sale', 'search_default_sale': 1}"
+            <field name="list_price" position="after">
+                <label for="taxes_id"/>
+                <div name="tax_info" class="o_row">
+                    <field name="taxes_id" widget="many2many_tags"
+                        context="{'search_default_sale': 1}"
```

Comment:
Looking into it, my guess is that `'default_type_tax_use': 'sale'` is being removed here because one cannot create a tax from this view. 
If so, please could you mention this in the commit message?

Reply:
Changed commit message,
Thanks.